src/MUIDataTable.js

```diff
@@ -935,6 +935,7 @@ class MUIDataTable extends React.Component {
           case 'multiselect':
             filterList[index] = value === '' ? [] : value;
             break;
+          case 'dropdown':
```

Comment:
It could become a source of subtle bugs to allow `dropdown` to fall through to `custom` here. If this is intended as a placeholder, I recommend a `break` and if not, I think it's ok to duplicate what's in `custom`.